faq/README.md

```diff
@@ -175,22 +175,20 @@ Next to that label, it will tell you the plugin version.
 
 ### Where are the log files saved?
 
-Go to Plugins > FFXIV Settings and look for the `Log file location`.
+Go to Plugins > FFXIV Settings and look for the `Log file location`. 
 
-### What is the difference between the ngld OverlayPlugin and hibiyasleep OverlayPlugin?
+The default location is `%appdata%\Advanced Combat Tracker\FFXIVLogs`. 
+If you have not customized the location, you can paste this into your start menu for quick access.
 
-* hibiyasleep's OverlayPlugin hasn't been updated in 2 years and thus uses an older version of Chromium to render
-  the overlays. That probably doesn't matter to you as a user but it means that overlays which work with the
-  hibiyasleep version can't use more recent features like modern ES JavaScript syntax, new CSS features, etc.
-* ngld's OverlayPlugin uses a single download for both x64 and x86 downloads to make installation easier.
-* ngld's OverlayPlugin integrates most of ACTWebSocket's features making the latter redundant.
-* ngld's OverlayPlugin added cutscene detection which automatically hides overlays during them. This feature
-  can be turned off in the plugin configuration if you don't want that.
-* ngld's OverlayPlugin allows you to assign the same hotkey to several overlays and doesn't require a restart
-  for hotkeys to become active.
-* ngld's OverlayPlugin offers more data for overlays.
-* For details check the [release notes](https://github.com/ngld/OverlayPlugin/releases) since they contain
-  the changes for each release.
+### Which OverlayPlugin fork am I supposed to use?
```

Comment:
Based on our conversation above, could you put this as the top question and update the TOC for the changes you made? I think that's my last feedback here.

Reply:
Done

Reply:
Did you disagree about making it the top question for more visibility? Or the fact that the general section is second, it doesn't matter if the top question in the second section?